### PR TITLE
BODS-3801

### PR DESCRIPTION
**Pull request by SamerIssam**
Add `First stop is set down only` to the timetable visualiser

### CODE REVIEW

File: transit_odp/browse/timetable_visualiser.py [hunk 1]
```diff
@@ -240,6 +248,56 @@ def get_df_servicedorg_vehicle_journey(
...
+    def get_observation_results_based_on_service_pattern_id(
+        self,
+        service_patter_ids: List,
+    ) -> dict:
+        """
+        Get the observation results based on the service pattern ids
+        and revision id
+        """
+        #TODO: This can be set via flag or a configuration
+        REQUIRED_OBSERVATIONS = "First stop is set down only"
```
Comment (NitishAroraKPMG) [inline transit_odp/browse/timetable_visualiser.py:260]: pass it as a parameter, the importance also

File: transit_odp/browse/timetable_visualiser.py [hunk 2]
```diff
@@ -240,6 +248,56 @@ def get_df_servicedorg_vehicle_journey(
...
+        REQUIRED_IMPORTANCE = "Critical"
+
+        columns = ["importance","observation" ,"service_pattern_stop_id", "vehicle_journey_id"]
+        #TODO: Avoid the hardcoding of the importance
```
Comment (NitishAroraKPMG) [inline transit_odp/browse/timetable_visualiser.py:264]: remove the comment and implement it

File: transit_odp/browse/timetable_visualiser.py [hunk 3]
```diff
@@ -240,6 +248,56 @@ def get_df_servicedorg_vehicle_journey(
 
         return pd.DataFrame.from_records(qs_serviced_orgs)
 
+    def get_observation_results_based_on_service_pattern_id(
+        self,
+        service_patter_ids: List,
```
Comment (NitishAroraKPMG) [inline transit_odp/browse/timetable_visualiser.py:253]: service_pattern_ids

File: transit_odp/browse/timetable_visualiser.py [hunk 4]
```diff
@@ -341,9 +398,16 @@ def get_timetable_visualiser(self) -> pd.DataFrame:
                     vehicle_journey_ids_non_operating
                 )
             ]
+            # Get the service pattern ids
+            service_pattern_ids = df_vehicle_journey_operating["service_pattern_stop_id"].unique().tolist()
```
Comment (NitishAroraKPMG) [inline transit_odp/browse/timetable_visualiser.py:402]: service_pattern_stop_ids

File: transit_odp/frontend/assets/js/tooltip.js [hunk 5]
```diff
@@ -6,8 +6,16 @@
  *
  * @example toggleTooltip('1', 'inbound') will add the class to show the tooltip
  */
-function showTooltip(event, rowIndex, direction) {
+function showTooltip(event, rowIndex, direction=null, type=null) {
   hideAllTooltips();
+  console.log(type);
+  console.log(direction);
```
Comment (NitishAroraKPMG) [inline transit_odp/frontend/assets/js/tooltip.js:12]: remove console.log

**Review by NitishAroraKPMG — COMMENTED** [review]
for now

**Review by pritish-ranjan01 — DISMISSED** [review]

**Review by NitishAroraKPMG — APPROVED** [review]

**Pull request merged**